packages/debug/src/browser/debug-session.tsx

```diff
@@ -73,8 +75,15 @@ export class DebugSession implements CompositeTreeElement {
         protected readonly breakpoints: BreakpointManager,
         protected readonly labelProvider: LabelProvider,
         protected readonly messages: MessageClient,
-        protected readonly fileService: FileService) {
+        protected readonly fileService: FileService,
+        protected readonly debugPluginCP: ContributionProvider<DebugPluginContribution>
```

Comment:
@amiramw 
could you give an example for what you mean?

Reply:
DebugPluginContribution is very specific for debug plugin flow. also it doesn't make sense that it is a list as there will always be one. it would be better if contribution is generic and plugin debug is only a specific implementation of it.

Reply:
done.